web/angular-wallet/src/app/app.component.ts

```diff
@@ -150,13 +153,22 @@ export class AppComponent extends UnsubscribeOnDestroy implements OnInit, OnDest
     try {
       const blockchainStatus = await this.networkService.getBlockchainStatus();
       this.isScanning = !this.firstTime && (this.previousLastBlock !== blockchainStatus.lastBlock);
```

Comment:
I like this: I was wondering how to differ between 'cold'-mode (full node) and 'hot'-mode (remote node)... pretty K.I.S.S. Nice one. In theory, if the remote node is not catched up, you would show this here, too.